src/NServiceBus.RabbitMQ/Connection/RabbitMqConnectionFactory.cs

```diff
@@ -44,10 +44,9 @@ public RabbitMqConnectionFactory(ConnectionConfiguration connectionConfiguration
             };
         }
 
-        public virtual IConnection CreateConnection(string purpose)
+        public virtual IConnection CreateConnection(string connectionName)
```

Comment:
[Every public change is a breaking change](https://blogs.msdn.microsoft.com/ericlippert/2012/01/09/every-public-change-is-a-breaking-change/) so there is always a class of theoretical breakages which need to be excluded from a project's definition of breaking, but I second the challenge. @bording what is the reason for making this change?